lib/PheWAS.js

```diff
@@ -1,6 +1,165 @@
-import React from 'react';
-import ChartPlaceholder from './ChartPlaceholder';
+import React, { Component } from 'react';
+import * as d3 from 'd3';
+import sortBy from 'lodash.sortby';
+import { withContentRect } from 'react-measure';
 
-const PheWAS = () => <ChartPlaceholder>PheWAS</ChartPlaceholder>;
+const SIGNIFICANCE = -Math.log10(5e-8);
+const DATA_POINT_RADIUS = 2;
 
-export default PheWAS;
+const margin = { top: 20, right: 30, bottom: 30, left: 40 };
+
+function hasAssociations(data) {
+  return (
+    data.pheWAS &&
+    data.pheWAS.associations &&
+    data.pheWAS.associations.length > 0
+  );
+}
+
+const x = d3.scaleBand();
```

Comment:
Better to have scales inside the PheWAS? If a using application created two PheWAS plots, they'd share a single pair of x, y scales.